diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -241,6 +244,9 @@ fun ASTNode.findAllNodesWithSpecificType(elementType: IElementType): List<ASTNod
  */
 fun ASTNode.isClassEnum(): Boolean = (psi as? KtClass)?.isEnum() ?: false
 
+fun ASTNode.isCompanionObject(): Boolean =
```

Comment:
There already is a method `isNodeFromCompanionObject`, why create a new one?

Reply:
Because this method returns true on property's level, but I should check if it is on class level

Reply:
Then you could simply call `KtObjectDeclaration#isCompanion` on `node.psi`.